src/server/tiered_storage.cc

```diff
@@ -22,10 +22,14 @@
 #include "server/tiering/common.h"
 #include "server/tiering/op_manager.h"
 #include "server/tiering/small_bins.h"
+#include "server/tx_base.h"
 
 ABSL_FLAG(bool, tiered_storage_cache_fetched, true,
           "WIP: Load results of offloaded reads to memory");
 
+ABSL_FLAG(size_t, tiered_storage_max_stashes, 50,
```

Comment:
`tiered_storage_max_stashes` -> ` tiered_storage_write_depth`,

the term used is queue depth, so `tiered_storage_write_depth` is more clear